src/components/Heatmap/Heatmap.jsx

```diff
@@ -11,13 +11,16 @@ import DotaMap from '../DotaMap';
  * Returns the adjusted heatmap data.
  */
 function scaleAndExtrema(points, scalef, max, shift) {
+  const averageValue = (points.reduce((a, b) => a + b.value, 0) / points.length);
```

Comment:
why are we using b.value but not a.value?

Reply:
ah you're right, I'm used to using reduce on arrays of numbers but this is an array of objects